libraries/launcher/org/prismlauncher/EntryPoint.java

```diff
@@ -136,6 +139,32 @@ private static ExitCode listen() {
         }
     }
 
+    private static void setProperties(Parameters params) {
+        String name = params.getString("instanceName", null);
+        String iconId = params.getString("instanceIconKey", null);
+        String iconPath = params.getString("instanceIconPath", null);
+        String windowTitle = params.getString("windowTitle", null);
+        String windowDimensions = params.getString("windowParams", null);
+
+        // set useful properties for mods
+        if (name != null)
+            System.setProperty("org.prismlauncher.instance.name", name);
+        if (iconId != null)
+            System.setProperty("org.prismlauncher.instance.icon.id", iconId);
+        if (iconPath != null)
+            System.setProperty("org.prismlauncher.instance.icon.path", iconPath);
+        if (windowTitle != null)
+            System.setProperty("org.prismlauncher.window.title", windowTitle);
+        if (windowDimensions != null)
+            System.setProperty("org.prismlauncher.window.dimensions", windowDimensions);
```

Comment:
I would honestly prefer to have a more generic naming scheme.

Maybe we should specify a standard format for these? That way we wouldn't need different implementations in different launchers.

Reply:
That's a pretty good idea

Reply:
I'm not sure on which namespace should be used, though.

Reply:
Here I would go with a more generic namespace like simply `launcher` or `minecraftlauncher`.
Another idea would be to make the namespace configurable by cmake, have a file like BuildConfig.cpp.in(but for java ) that is populated by cmake, and then we can put that as Launcher_Domain but reversed.

Reply:
We could also form a compromise and assert ourselves as a standard.

Use `minecraft.launcher.brand` and `minecraft.launcher.version` and add our own with `org.prismlauncher`

while we are at it, let's also revert https://github.com/PolyMC/PolyMC/pull/606

Reply:
nah, I think that change was OK.

Reply:
> nah, I think that change was OK.

It was free advertising though :3